src/index.ts

```diff
@@ -50,16 +56,22 @@ app.post("/upload-uidl", async (req, res) => {
   }
 });
 
-app.get("/fetch-uidl/:fileName", async (req, res) => {
-  const { fileName } = req.params;
+app.get("/fetch-uidl/:fileName/:type", async (req, res) => {
```

Comment:
I think is a better idea to switch filename and type in order to have /fetch-uidl/:type/:fileName. So type will be like a namespace. Also we should check if type is neither equal to "component" or "project"

Reply:
I thought the same, but the problem is these API's are being used in our 3rd party projects. So, we need to change in all those places too. But still seems reasonable. We can change and see what all projects needs change.